src/main/java/mate/academy/bookshop/util/IsbnFormatter.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.bookshop.util;
+
+public class IsbnFormatter {
+    private static final int STANDARD_BOOK_NUMBER_LENGTH = 9;
+    private static final int CHECK_NUMBER_LENGTH = 1;
+
+    public static String format(String isbn) {
+        char[] c = isbn.toCharArray();
+
+        StringBuilder builder = new StringBuilder();
+        for (int i = c.length - 1 - CHECK_NUMBER_LENGTH; i >= 0; i--) {
+            if (Character.isDigit(c[i])) {
+                builder.append(c[i]);
+                if (builder.length() == STANDARD_BOOK_NUMBER_LENGTH) {
+                    break;
+                }
+            }
+        }
+        return builder.reverse().toString();

```

Comment:
honestly, I can't imagine why would you need that

Reply:
I changed this class now, this is necessary so that ISBNs are stored in the database in a single form, for example "0-596-52068-9" == "ISBN-10: 0-596-52068-9", but if we do not format them then for the database they will be different